reactor-netty-core/src/main/java/reactor/netty/transport/NameResolverProvider.java

```diff
@@ -627,6 +643,14 @@ public NameResolverSpec resolvedAddressTypes(ResolvedAddressTypes resolvedAddres
 			return this;
 		}
 
+		@Override
+		public NameResolverSpec localAddress(Supplier<? extends SocketAddress> bindAddressSupplier)	{
+			if (bindAddressSupplier != null) {
```

Comment:
The idea of the `Supplier` is to create the `SocketAddress` lazy. Please do not obtain the `SocketAddress` here. The field `localAddress` will be `Supplier` and not `SocketAddress`.
Check only that the `Supplier` is not null. Later when we configure the name resolver we will handle the case when the `Supplier` returns null.

Reply:
done